src/send.js

```diff
@@ -32,8 +35,14 @@ let sendDataPoints = async (configurer, options, payload) => {
       },
       encoding: null,
       body:     serialize(payload),
-      timeout:  30 * 1000,
-      retries:  5,
+      timeout:  90 * 1000,
+      retries:  (iter, error) => {
```

Comment:
If we experience 7 timeouts in a row, this would keep trying for ~10 minutes right? I also wonder if we still want to ignore ENETUNREACH. That feels like it definitely wouldn't be transient?